Fix pyenv FAQ env var commands (#144)
brettcannon/python-launcher: README.md

### How can I make the Python Launcher use my default Python version from pyenv?

If you're using [pyenv](https://github.com/pyenv/pyenv) to manage your Python

> versions, you'll want to set the version the Launcher to the pyenv 

Reviewer: ```suggestion
versions, you'll want to set the version the Launcher uses to the pyenv 
```

[global version](https://github.com/pyenv/pyenv/blob/master/COMMANDS.md#pyenv-global).

> Add these lines to your `.zshrc` or `.bashrc` file:

Reviewer: ```suggestion
Add this line to your `.zshrc` or `.bashrc` file:
```

```sh
export PY_PYTHON=$(pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[:2])))")
```

> Or these lines to your `~/.config/fish/config.fish` file:

Reviewer: ```suggestion
Or this line to your `~/.config/fish/config.fish` file:
```

### How can I make the Python Launcher use my default Python version from pyenv?

If you're using [pyenv](https://github.com/pyenv/pyenv) to manage your Python
versions, you'll want to set the version the Launcher to the pyenv 
[global version](https://github.com/pyenv/pyenv/blob/master/COMMANDS.md#pyenv-global).

Add these lines to your `.zshrc` or `.bashrc` file:

```sh

> export PY_PYTHON=$(pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[0:2])))")

Reviewer: ```suggestion
export PY_PYTHON=$(pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[:2])))")
```

> export PY_PYTHON=$(pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[0:2])))")
> export PY_PYTHON3=$(pyenv exec python3 -c "import sys; print(*sys.version_info[0:2], sep='.')")

Reviewer: ```suggestion
export PY_PYTHON=$(pyenv exec python3 -c "import sys; print(*sys.version_info[0:2], sep='.')")
```

> export PY_PYTHON3=$(pyenv exec python3 -c "import sys; print(*sys.version_info[0:2], sep='.')")

Reviewer: ```suggestion
```

```

Or these lines to your `~/.config/fish/config.fish` file:

```sh

> set -gx PY_PYTHON (pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[0:2])))")

Reviewer: ```suggestion
set -gx PY_PYTHON (pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[:2])))")
```

> set -gx PY_PYTHON (pyenv exec python -c "import sys; print('.'.join(map(str, sys.version_info[0:2])))")
> set -gx PY_PYTHON3 (pyenv exec python3 -c "import sys; print(*sys.version_info[0:2], sep='.')")

Reviewer: ```suggestion
set -gx PY_PYTHON (pyenv exec python3 -c "import sys; print(*sys.version_info[0:2], sep='.')")
```

> set -gx PY_PYTHON3 (pyenv exec python3 -c "import sys; print(*sys.version_info[0:2], sep='.')")

Reviewer: ```suggestion
```
